packages/dd-trace/test/appsec/iast/utils.js

```diff
@@ -123,8 +123,10 @@ function endResponse (res, appResult) {
       res.end()
     })
   } else {
-    res.writeHead(200)
-    res.end()
+    if (!res.headersSent) {
+      res.writeHead(200)
+      res.end()
+    }
```

Comment:
You can move this `res.end` out of the if, at this way you will prevent the writeHead when it is not necessary, but if some test writes some header and doesn't end the response, it will finish the response.
Sending `res.end()` twice doesn't break the app.

```suggestion
    }
    res.end()
```